examples/applications/plot_out_of_core_classification.py

```diff
@@ -70,6 +72,9 @@ def __init__(self, encoding='latin-1'):
 
     def handle_starttag(self, tag, attrs):
         method = 'start_' + tag
+        for attr in attrs:
+            if attr[0] == 'lewissplit':
+                self.LEWisSplit = attr[1]
```

Comment:
why this capitalisation?

Reply:
As per guidance to split train / test datasets, "LEWISSPLIT" is attribute that achieve the same. So, my purpose to express this variable in this manner is, Learners are easily identify this attribute.

Reply:
`LEWisSplit` is not a conventional attribute name. Is `lewis_split` appropriate?

Reply:
Yes, it's appropriate